me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/IDmeWebVerify.java

```diff
@@ -204,34 +200,49 @@ private String ExtractAccessToken(String url)
      */
     private void GetWebProfile(String url)
     {
+        String serverResponse;
+        HttpURLConnection urlConnection;
+        URL urlRequest;
+        try {
+            urlRequest = new URL(url);
+            urlConnection = (HttpURLConnection) urlRequest.openConnection();
+
+            int responseCode = urlConnection.getResponseCode();
```

Comment:
I see that this is blocking. Is this right? Maybe changing it for another PR, but maybe this shouldn't be blocking.